src/include/wiredtiger.in

```diff
@@ -1659,6 +1659,18 @@ struct __wt_session {
 	 */
 	int __F(prepare_transaction)(WT_SESSION *session, const char *config);
 
+	/*!
+	 * Reset the snapshot.
+	 *
+	 * This method resets snapshots for snapshot isolation transactions to
+	 * update their existing snapshot. It raises an error for when this API
```

Comment:
How about `It raises an error when this API is used other than snapshot isolation mode`?

Reply:
Yeah, fixed the comment section.